### ISSUE TITLE
All my extensions are not working

### ISSUE THREAD

**Issue by varchasvipandey** [microsoft/vscode#94756]
Issue Type: <b>Bug</b>

Extension host terminated unexpectedly is popped as the first message. All the plugins including some default IntelliSense plugins are not working. This is making my workflow very slow. Please fix this asap. 

VS Code version: Code 1.44.0 (2aae1f26c72891c399f860409176fe435a154b13, 2020-04-07T23:31:18.860Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 2600 Six-Core Processor             (12 x 3394)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off_ok<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.93GB (2.55GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (35)</summary>

Extension|Author (truncated)|Version
---|---|---
jest-snippets|and|1.8.0
code-gnu-global|aus|0.2.2
vscode-intelephense-client|bme|1.3.11
path-intellisense|chr|1.4.2
es7-react-js-snippets|dsz|2.7.1
tslint|eg2|1.0.44
eppz-code|epp|1.2.41
vsc-material-theme|Equ|32.5.0
vsc-material-theme-icons|equ|1.1.4
prettier-vscode|esb|4.1.1
php-debug|fel|1.13.0
php-intellisense|fel|2.3.14
php-pack|fel|1.0.2
code-runner|for|0.9.17
terminal|for|0.0.10
sublime-babel-vscode|jos|0.2.10
node-module-intellisense|lei|1.5.0
vscode-language-babel|mgm|0.0.27
csharp|ms-|1.21.16
python|ms-|2020.3.71659
cpptools|ms-|0.27.0
vscode-typescript-tslint-plugin|ms-|1.2.3
debugger-for-chrome|msj|4.12.6
vscode-react-native|msj|0.14.2
laravel-blade|one|1.20.0
vscode-jest|Ort|3.1.1
material-icon-theme|PKi|4.0.1
tsimporter|pmn|2.0.1
live-sass|rit|3.0.0
LiveServer|rit|5.6.1
mongo-snippets-for-node-js|roe|1.2.0
js-jsx-snippets|sky|10.1.0
unity-debug|Uni|2.7.5
vscode-icons|vsc|10.0.0
JavaScriptSnippets|xab|1.7.2

(6 theme extensions excluded)

</details>

**Comment by gjsjohnmurray** [comment]
Please find out which of your extensions is causing the issue. Either disable all and then re-enable one at a time, or disable half of them to narrow the field faster (bisecting).

**Comment by varchasvipandey** [comment]
Disabled all the extensions but nothing happened. Still facing the same
problem.

**Comment by gjsjohnmurray** [comment]
You disabled all your extensions, then restarted VS Code, and you still got a message about "Extension host terminated unexpectedly", right?

Any more information in that message? Maybe paste a screenshot here?

**Comment by varchasvipandey** [comment]
I figured it out. The reason is the latest update 1.44
I reinstalled the software, I have version 1.38 installer. Everything
became normal until I closed the application and saw updating VS code
popped up on my screen. I again launched the application and the problem
was there again. I again uninstalled this version and reinstalled it with
v1.38 installer. This time before launching the application I turned off my
Internet connection. Then I headed over to the settings and disabled
update. Re enabled my extensions and my problem was thus solved.

Conclusion:
Problem is with the latest update for Windows i.e.  1.44
